server/services/courtRegisterService.ts

```diff
@@ -47,6 +65,14 @@ export default class CourtRegisterService {
     await CourtRegisterService.restClient(token).put({ path: `/court-maintenance/id/${courtId}`, data: updatedCourt })
   }
 
+  async addCourt(context: Context, addCourt: AddCourt): Promise<AddUpdateResponse> {
+    // TODO - add service calls
+    logger.info(`Adding court ${addCourt.court.courtId}`)
+    return new Promise<AddUpdateResponse>(resolve => {
+      resolve({ success: true })
+    })
+  }
```

Comment:
think this is the equivalent of `return { success: true }`? 

doesn't a return from an async method always wrap in a promise?

Reply:
Yes, correct. Thanks